xml/depl_nodes.xml

```diff
@@ -2275,6 +2315,92 @@ getent group postgres | cut -d: -f3</screen>
    </varlistentry>
   </variablelist>
 
+  <bridgehead renderas="sect2"><guimenu>Hitachi HUSVM</guimenu>
+  </bridgehead>
+
+  <variablelist>
+    <varlistentry>
+      <term>Storage Protocol</term>
+      <listitem>
+      <para>
+	Choose between the FibreChannel and iSCSI protocols.<remark>2016-12-29 - dpopov: DEVS FIXME Please check whether this is correct.</remark>
+      </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term>Range of group number</term>
+      <listitem>
+      <para>
+	None <remark>2016-12-29 - dpopov: DEVS FIXME Please provide info.</remark>
+      </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term>Add to HORCM configuration</term>
+      <listitem>
+      <para>
+	Choose <systemitem>true</systemitem> to add to the HORCM (Hitachi Online Remote Copy Manager) configuration.<remark>2016-12-29 - dpopov: DEVS FIXME Please check whether this is correct.</remark>
+      </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term>Username/Password of storage system for HORCM</term>
+      <listitem>
+      <para>
+	Login credentials for HORCM.
+      </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term>Range of logical device of storage system</term>
+      <listitem>
+      <para>
+	None<remark>2016-12-29 - dpopov: DEVS FIXME Please provide info.</remark>
+      </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term>Pool ID of storage system</term>
+      <listitem>
+      <para>
+	None<remark>2016-12-29 - dpopov: DEVS FIXME Please provide info.</remark>
+      </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term>Serial number of storage system</term>
+      <listitem>
+      <para>
+	None<remark>2016-12-29 - dpopov: DEVS FIXME Please provide info.</remark>
```

Comment:
this comes from http://docs.openstack.org/newton/config-reference/block-storage/drivers/hitachi-storage-volume-driver.html which doesn't provide more information. My suggestion would be to link the upstream config reference instead of trying to come up with better documentation here. this applies to all hitachi options.  We can much quicker fix that upstream than in our documentation, and then it benefits everyone.